client/pages/sign-up.vue

```diff
@@ -0,0 +1,130 @@
+<template>
+  <AuthForm v-if="!accountCreated" v-slot="{ invalid }" @submit="register">
+    <AuthAlert :errors="errors">
+      Error creating account, please fix the following errors:
+    </AuthAlert>
+    <InputField
+      v-for="field in fields"
+      :id="field.id"
+      :key="field.index"
+      :field-name="field.name"
+      :field-type="field.type"
+      :field-options="field.options"
+      :rules="field.rules"
+      :inputvalue.sync="field.value"
+    />
+    <AuthSubmit :disabled="invalid">Submit</AuthSubmit>
+    <p class="self-center mt-3">
+      Already have an account?
+      <NuxtLink to="/login" class="text-blue">Click here to log in</NuxtLink>
+    </p>
+  </AuthForm>
+  <AuthForm
+    v-else
+    class="place-items-center gap-5 font-bold text-2xl text-white"
+  >
+    <h1 class="text-3xl">Congratulations {{ name }}!</h1>
+    <p>
+      Your new Elucidate account<wbr />
+      has been created
+    </p>
+    <font-awesome-icon
+      :icon="['fas', 'fa-face-smile-beam']"
+      class="text-[10rem] my-10"
+    />
+    <p>
+      Please proceed to the
```

Comment:
We should be redirecting to the login page if registration was successful.

We should add something to the bottom of the `register` function like: 
```
this.$router.push("/login")
```
If we have successfully signed up

Reply:
Agreed. Alternatively, we could sign in programmatically and redirect to the dashboard.

Reply:
@jsun1590 Should we do this instead of following the figma?

Reply:
I don't have a mind either, but personally I think having a "successful sign up" page looks good

Reply:
I'll keep the successful sign up page because you & the figma design agree on it.
Probably better for the client to get it as it was shown to them.